docs/DataWorkflows/FetchingYourData.md

```diff
@@ -95,7 +93,7 @@ In order to get the data to your computer, you need to ensure 3 things:
 
 If these conditions are met, you're ready to access some data! Suppose I'd like
```

Comment:
For more clarity, I'd break this with a subtitle up in the following way: 
If these conditions are met, you're ready to access some data!
Generally, datasets can be in two different formats: 
- **unzipped**: this type of dataset allows to access all different subdirectories and files of all participants directly
- **zipped**: this type of dataset has one zip file per participant. This allows more efficient storing, and requires a few extra steps to figure out what kinds of files are available and to download them. 

Ways of looking at and then getting each type of dataset are explained below.

## Getting an unzipped dataset
Suppose I'd like to see the regional time series data for CCNP.
[etc.]